src/components/jobbonsker.tsx

```diff
@@ -60,14 +66,49 @@ const Jobbonsker = () => {
         );
     }
 
+    if (!underOppfolgingData?.underOppfolging) {
+        return (
+            <Panel border className="info_panel" tabIndex={5}>
```

Comment:
Fjerne tabindex da vi ikke skulle ha dette på kortene (gjelder flere steder i koden):

```suggestion
            <Panel border className="info_panel">
```